src/pages/_document.page.jsx

```diff
@@ -0,0 +1,18 @@
+import {
+  Html,
+  Head,
+  Main,
+  NextScript,
+} from 'next/document';
+
+export default function Document() {
+  return (
+    <Html lang="en">
+      <Head />
```

Comment:
I left this in a different comment, but what doe this file actually do? Just trying to make sure I understand!

Reply:
It's basically creating a template for the html structure, in this case I'm using it to add the lang attribute to the html element.